### DISCUSSION TITLE
OSCAL DEFINE - Research - March 16, 2023

### DISCUSSION THREAD

**Discussion by Compton-US** [usnistgov/OSCAL-DEFINE#11] [Community Meetings]
> Call details are located here: https://pages.nist.gov/OSCAL/contribute/define-meeting/

These meetings are open to the public, and this discussion post is open for feedback regarding topics discussed at the meeting.

### Topics to be discussed:

- An overview of the current workflow.
- The first research effort: https://github.com/usnistgov/OSCAL-DEFINE/issues/5
  - Current spiral for the effort: https://github.com/usnistgov/OSCAL-DEFINE/issues/10
  - Spiral work in progress: https://github.com/usnistgov/OSCAL-DEFINE/pull/8

#### Supporting Issues

- Use Case Scenarios:  	
  - (Spiral 1) Research use of a Customer Responsibility Matrix (CRM) model for control element content. 
  - ([#1336](https://github.com/usnistgov/OSCAL/issues/1336))

- Gap Analysis: 		
  - (Spiral 2) Map use case information to existing OSCAL model(s) and identify gaps. 
  - ([#1385](https://github.com/usnistgov/OSCAL/issues/1385))

- Export Examples:	
  - (Spiral 3) Create Examples of SSP Exports to Support Creation of Responsibility Model 
  - ([#1467](https://github.com/usnistgov/OSCAL/issues/1467))

### Opportunities to get involved:

- Responsibility model draft specification: https://github.com/usnistgov/OSCAL/discussions/1695
- OSCAL profile tutorial: https://github.com/usnistgov/OSCAL/discussions/1696

**Comment by Compton-US** [comment]
Today's presentation slides:

[OSCAL DEFINE 2023-03-16.pdf](https://github.com/usnistgov/OSCAL-DEFINE/files/10992943/OSCAL.DEFINE.2023-03-16.pdf)

**Comment by Compton-US** [comment]
Suggestions were made to consider an `export` attribute for exportable content to support larger scale numbers of controls.  This could use structured data fields (not sharing remarks or descriptions), and also use the existing descriptions under exports where sharable descriptions are desired.

This should be considered as a part of the draft specification effort.

**Reply by aj-stein-nist** [reply]
So if we have questions about an item here that was discussed in the meeting and next steps, would you prefer I ask here or on issue #5?

**Reply by Compton-US** [reply]
@aj-stein-nist At this point, either place is acceptable.  If it is specific to comments or slides above, just add here.

**Comment by Compton-US** [comment]
One concept that I was looking for comment on was the addition of an export assembly to the component definition.  I've heard mostly comments against doing this but welcome input here for consideration.
